src/pages/Profile.js

```diff
@@ -62,106 +66,142 @@ export default function Profile(props) {
-                  <IconButton
-                    target={"_blank"}
-                    href={profile.connections[1].facebook}
-                    aria-label="twitter"
-                  >
-                    <Facebook />
-                  </IconButton>
-                ) : null}
-                <ShareButton />
-              </Container>
-            ) : null}
-          </Container>
-          <Divider variant="middle" style={{ marginTop: 30 }} />
-          <Container>
-            <Grid container spacing={3}>
-              <Grid
-                item
-                xs
-                style={{
-                  marginTop: 20,
-                  display: "flex",
-                  justifyContent: "center",
-                }}
-              >
+            <Grid container spacing={2}>
+              <Grid item>
                 {!isLoading ? (
                   <Avatar
                     style={{ height: 175, width: 175 }}
                     aria-label={"Author"}
-                    src={profile.avatar ? profile.avatar[0].url : null}
+                    src={
+                      profile.avatar.length >= 1 ? profile.avatar[0].url : null
+                    }
                   />
-                ) : null}
-              </Grid>
-              <Grid item xs={6}>
-                {!isLoading ? (
-                  postFeed.map((item, index) => (
-                    <BlogPreviewProfile key={item.id} post={item} />
-                  ))
                 ) : (
                   <Skeleton
-                    height={window.innerHeight * 0.75}
+                    variant={"circle"}
                     animation={"wave"}
+                    height={175}
+                    width={175}
                   />
                 )}
               </Grid>
-              <Grid item xs></Grid>
+              <Grid
+                item
+                xs={12}
+                sm
+                container
+                style={{
+                  marginLeft: 14,
+                  display: "flex",
+                  alignItems: "center",
+                }}
+              >
+                <Container>
+                  <Typography variant={"h3"}>
+                    {!isLoading ? (
+                      displayName(profile, id.substring(8))
+                    ) : (
+                      <Skeleton animation={"wave"} />
+                    )}
+                  </Typography>
+                  <Typography
+                    variant={"h6"}
+                    gutterBottom={true}
+                    color={"textSecondary"}
+                  >
+                    {!isLoading ? (
+                      profile.aboutMe
+                    ) : (
+                      <Skeleton animation={"wave"} />
+                    )}
+                  </Typography>
+                </Container>
+
+                {!isLoading && profile.connections.length > 0 ? (
+                  <Container>
```

Comment:
follow up: if these icons are used in multiple places you could refactor them into a `socialicons.js` file and have a function that takes the `profile.connections` and returns the appropriate elements. 

Might be worth doing anyways.